README.md

```diff
@@ -65,6 +65,29 @@ The app automatically launches with the Chromium developer tools opened by defau
 
 Code formatting has been set up to make merging PRs easier. It uses the same prettier/eslint mechanism as Calypso. See [JavaScript Coding Guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/javascript.md) for details on setting up your editor.
 
+### Testing
+
+#### Running tests
+
+You can run tests with the following command:
+
+```bash
+npm run test
+```
+
+There are also E2E tests available. They require a little bit more setup - clean the `out/` directory and build the app binary: 
+
+```bash
+rm -rf out/
```

Comment:
Do we need to include the slash `/` in the path? If not, we could remove it and avoid conflicts with Windows.

Reply:
I'm also thinking whether we explicitly need to remove this folder. Wouldn't be enough running command `npm run make` to generate a new build?

Reply:
> Do we need to include the slash / in the path? If not, we could remove it and avoid conflicts with Windows.

Without the slash, it throws the error, but it seems that it would be more correct to ask the user to empty the directory e.g. `rm -rf out/*`. What if we dropped the `rm` instruction completely and just asked user to empty the `out/` directory?

Reply:
Another option would be to have a `pre` script:
```
"pree2e": "npm run make && npx playwright install",
"e2e": "npx playwright test",
```

This way when running `npm run e2e`, we ensure that the preparation steps are run. If you only need to run a single test, you can do `npm run e2e --ignore-scripts -- <test-filter>` (e.g. `npm run e2e --ignore-scripts -- app.test`).

Reply:
> I'm also thinking whether we explicitly need to remove this folder. Wouldn't be enough running command npm run make to generate a new build?

I proposed to empty the `out/` directory as E2E tests fail if a user built the x64 on Mac Silicon before (this is what I did).

Reply:
I removed that https://github.com/Automattic/studio/pull/190/commits/3779c16f99ddd54bde4797045dc57226afb747ba

Reply:
> Without the slash, it throws the error, but it seems that it would be more correct to ask the user to empty the directory e.g. `rm -rf out/*`.

I'm not experiencing any error when running `rm -rf out`, what error are you encountering? On the other hand, this command won't work on Windows, so we either ask the user to remove it (i.e. without sharing the command) or include instructions for Windows.

> What if we dropped the `rm` instruction completely and just asked user to empty the `out/` directory?

Yeah, or ask the user to remove the `x64` folder if that's what causes the issue. I presume on Windows this won't be an issue, hence we could specify that the problem only affects macOS.

> I proposed to empty the `out/` directory as E2E tests fail if a user built the x64 on Mac Silicon before (this is what I did).

Ah, ok. So removing the folder would be only required to avoid using the wrong binary 👍 .